### DISCUSSION TITLE
Specs for AppWindow, EnvironmentManager and PowerManager (or more?) are missing in /specs. Is this omission intentional?

### DISCUSSION THREAD

**Discussion by roxk** [microsoft/WindowsAppSDK#2423] [General]
As title.

**Comment by DrusTheAxe** [comment]
More like (mostly) a side effect of early days of the project getting rolling. Some early features were spec'd using Windows' traditional tools and processes and not per the current WinAppSDK process (markdown docs under `\specs`, etc). I'll ping a few folks and see if we can backfill `\specs` for ~posterity~ future reference ;-) but there may be reasons some are ill suited to be published.

Going forward this generally shouldn't occur. WinAppSDK's policy is to default to open/public specs and code, unless there's good reason to be closed/private (e.g. IP or like secrets that cannot be disclosed).

Was there something in particular you were looking for or you happened to notice the gap and inquired?

**Reply by roxk** [reply]
Thanks for the explanation.

I'd like to read the spec on `AppWindow`, only to find out they are missing. I was investigating why certain title bar customization is only available on windows 11, in the hope that maybe the community can find a way to back port it to windows 10 (the polyfill promise is important).
